Orm/Xtensive.Orm/Orm/Logging/IndentManager.cs

```diff
@@ -26,35 +26,35 @@ public void Dispose()
           return;
         disposed = true;
         CurrentIndentValueAsync.Value = oldIndent;
-        if (endAction!=null)
-          endAction.Invoke();
+        endAction?.Invoke();
       }
 
-      public IndentScope(string oldIndent, Action endAction)
+      public IndentScope(int oldIndent, Action endAction)
       {
         this.oldIndent = oldIndent;
         this.endAction = endAction;
+        disposed = false;
       }
     }
 
-    private const string SingleIndent = "  ";
+    private const int SingleIndent = 2;
     
-    private static readonly AsyncLocal<string> CurrentIndentValueAsync = new AsyncLocal<string>();
+    private static readonly AsyncLocal<int> CurrentIndentValueAsync = new();
 
     /// <summary>
     /// Gets indentation for current thread.
     /// </summary>
-    public static string CurrentIdent { get { return CurrentIndentValueAsync.Value ?? string.Empty; } }
+    public static int CurrentIdent => CurrentIndentValueAsync.Value;
```

Comment:
The same here, lets rename it to CurrentIndentLength or something like that, at least typo should be fixed since IndentManager is changing.

Reply:
done